Sofa/Sofa/League Matches/Event/EventViewModel.swift

```diff
@@ -53,4 +53,23 @@ class EventViewModel {
         let startDate = Date(timeIntervalSince1970: TimeInterval(event.startTimestamp))
         return formatter.string(from: startDate)
     }
+    
+    var homeTeamViewModel: EventTeamViewModel {
+        EventTeamViewModel(
+            name: event.homeTeam.name,
+            logoUrl: event.homeTeam.logoUrl,
+            score: event.homeScore != nil ? "\(event.homeScore!)" : "",
```

Comment:
Bez forceUnwrapanja!

Reply:
Slobodno pošalji nil vrijednost, nek bude score optional, možeš na label.text postaviti nil vrijednost, UILabel to zna handlati.